scriptmodules/libretrocores/lr-pcsx-rearmed.sh

```diff
@@ -48,7 +48,7 @@ function configure_lr-pcsx-rearmed() {
     ensureSystemretroconfig "psx"
 
     # system-specific, PSX
-    iniConfig " = " "" "$configdir/psx/retroarch.cfg"
+    iniConfig " = " '"' "$configdir/psx/retroarch.cfg"
     iniSet "rewind_enable" "false"
```

Comment:
good point. - been broken for ages then. we should remove it maybe - or do a "insert before" ini thing